app/walletmediums/dcrwalletrpc/dcrwalletrpc.go

```
@@ -74,6 +136,13 @@ func New(ctx context.Context, rpcAddress, rpcCert string, noTLS bool) (*WalletRP
 	}
 }
 
+func saveRPCAddress(address string) {
+	err := config.UpdateConfigFile("walletrpcserver", address, true)
+	if err != nil {
+		fmt.Printf("unable to save rpc address: %s", err.Error())
```

Printing this error would have been necessary if the user was prompted for an address and saving it failed. Since we're autodetecting, ignore this error. That would also mean that this function can be removed in favor of simply using `config.UpdateConfigFile("walletrpcserver", address, true)` in the call sites